easybuild/easyconfigs/j/Java/Java-11.0.16.eb

```diff
@@ -1,28 +1,28 @@
 name = 'Java'
-version = '11.0.16'
-local_build = '8'
+_java_version = '11'
+_patch_version = '16'
+_build_version = '8'
+version = f'{_java_version}.0.{_patch_version}'
 
 homepage = 'http://openjdk.java.net'
 description = """Java Platform, Standard Edition (Java SE) lets you develop and deploy
 Java applications on desktops and servers."""
 
 toolchain = SYSTEM
 
-local_tarball_tmpl = 'OpenJDK%%(version_major)sU-jdk_%s_linux_hotspot_%%(version)s_%s.tar.gz'
-
 # Using the Adoptium Eclipse Temurin builds, recommended by https://whichjdk.com/#distributions
-
-source_urls = ['https://github.com/adoptium/temurin%%(version_major)s-binaries/releases/download/jdk-%%(version)s+%s/'
-               % local_build]
-sources = [local_tarball_tmpl % ('%(jdkarch)s', local_build)]
-
+_tarball_tmpl = f'OpenJDK{_java_version}U-jdk_{{}}_linux_hotspot_{version}_{_build_version}.tar.gz'
+sources = [_tarball_tmpl.format('%(jdkarch)s')]
+source_urls = [
```

Comment:
@lexming We usually have `source_urls` above `sources`, why need keep doing that?

Reply:
yeah I know, but scripting these changes late at night is hard :slightly_smiling_face: